CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ Current Trunk
 -------------
 - Remove asm2wasm, which supported Emscripten's fastcomp backend, after fastcomp
   was removed.
+- Enabling the exception handling feature now requires enabling the reference
+  types feature as well since `exnref` depends on it. The same applies to the
+  new anyref feature where `anyref` depends on reference types.
```

Comment:
This has been the case before (that you have to specify dependent features too), so nothing has changed here. If we later add feature implying feature then we need to write something here.

But it'd be worth noting that `--enable-anyref` is added, and what it does.

Reply:
Added this because it is a validation error now (as of the current state of this PR), but wasn't before. Previously one would only get a validation error further down the road if `--enable-exception-handling` was specified, an `exnref` emitted somehow but `--enable-reference-types` omitted.

Reply:
Split the entry into two parts, with one mentioning the new feature and one mentioning that not enabling implied features is a validation error now.

Reply:
Oh sorry, I misinterpreted the comment. I actually forgot that this PR makes it a validation error if dependent flags aren't specified. I was not in favor of that, but maybe we should do feature implication later.